src/main/resources/META-INF/rewrite/junit5.yml

```diff
@@ -79,6 +79,7 @@ recipeList:
   - org.openrewrite.java.testing.junit5.UpdateBeforeAfterAnnotations
   - org.openrewrite.java.testing.junit5.LifecycleNonPrivate
   - org.openrewrite.java.testing.junit5.UpdateTestAnnotation
+  - org.openrewrite.java.testing.junit5.AddMissingTestBeforeAfterAnnotations
```

Comment:
I worry that we might be sensitive to ordering issues here, where if the super class has it's annotations changed before the extending class, would we then fail to match given that our predicate only checks for the old annotations?

Reply:
see below